app/templates/main/map.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
+<script type="text/javascript" src="https://maps.google.com/maps/api/js?sensor=true"></script>
+<script type="text/javascript" src="http://hpneo.github.io/gmaps/gmaps.js"></script>
+    <style type="text/css">
+        #map {
```

Comment:
i fixed everything but the sass part because I wasn't sure how I'd reference the other sheet & how to format it in sass -- could we discuss during the meeting tomorrow?